tests/conftest.py

```diff
@@ -131,9 +131,9 @@ def fake_snapd():
     socket_path_patcher = mock.patch(
         "craft_parts.packages.snaps.get_snapd_socket_path_template"
     )
+    escaped_path = snapd_fake_socket_path.replace("/", "%2F")
     mock_socket_path = socket_path_patcher.start()
-    mock_socket_path.return_value = f'\
```

Comment:
this is unrelated but pyright started complaining about an "unterminated string" here